### PR TITLE
Fix RuboCop lint offenses

### PR DESCRIPTION
**Pull request by agrimm**
Fix most of the offenses reported by RuboCop when run with the `--lint` option.

This was done so that when this gem is used while warnings are turned on (`RUBYOPT="-w"`), this gem produces fewer warnings.

The gem will still produce warnings about uninitialized instance variables being retrieved by monetizable.rb

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ end
 
 APP_RAKEFILE = File.expand_path("../spec/dummy/Rakefile", __FILE__)
 
-load 'rails/tasks/engine.rake' if File.exists?(APP_RAKEFILE)
+load 'rails/tasks/engine.rake' if File.exist?(APP_RAKEFILE)
```
Comment (antstorm) [inline Rakefile]: What's the reason for this change?

File: lib/money-rails/active_record/monetizable.rb [hunk 2]
```diff
@@ -85,10 +85,10 @@ def monetize(*fields)
             #
             # To disable validation entirely, use :disable_validation, E.g:
             #   monetize :price_in_a_range_cents, :disable_validation => true
-            if validation_enabled = MoneyRails.include_validations && !options[:disable_validation]
+            if (validation_enabled = MoneyRails.include_validations && !options[:disable_validation])
```
Comment (antstorm) [inline lib/money-rails/active_record/monetizable.rb]: Hmm… not sure about this one. Is it because of the assignment inside a condition?

File: Rakefile (hunk 1 above)
Comment (agrimm) [inline Rakefile]: File.exists? is deprecated - it generates a warning when Ruby is run in verbose mode from Ruby 2.1.0 onwards.

File: lib/money-rails/active_record/monetizable.rb (hunk 2 above)
Comment (agrimm) [inline lib/money-rails/active_record/monetizable.rb]: Yes. This one doesn't generate a Ruby warning if I recall correctly, but I thought I may as well fix all the RuboCop lint offenses in one go, rather than in dribs and drabs. If you don't want this as part of the commit that's fine though.

File: Rakefile (hunk 1 above)
Comment (antstorm) [inline Rakefile]: Oh, didn't know that

File: lib/money-rails/active_record/monetizable.rb (hunk 2 above)
Comment (antstorm) [inline lib/money-rails/active_record/monetizable.rb]: I think we should follow the community style guide — https://github.com/bbatsov/ruby-style-guide#no-parens-if

And it seems like this is a default option for RuboCop

Comment (agrimm) [inline lib/money-rails/active_record/monetizable.rb]: "Note that there is an exception to this rule, namely safe assignment in condition." https://github.com/bbatsov/ruby-style-guide#safe-assignment-in-condition

Comment (antstorm) [inline lib/money-rails/active_record/monetizable.rb]: Hmm, ok, fair enough

**Comment by antstorm** [comment]
That looks good to go. Thanks, @agrimm!
